packages/component/src/hooks/useEmojiFromStyles.js

```diff
@@ -0,0 +1,51 @@
+/*eslint no-useless-escape: "off"*/
+/*eslint require-unicode-regexp: "off" */
```

Comment:
Why these are off?

Reply:
`/` not needing an escape makes me nervous, I guess... removed the escape.

For the second one, I can't the linting error to go away, even when 'u' is added. So I added the comment. Am I missing something?
![image](https://user-images.githubusercontent.com/14900841/87755516-fc10e980-c7bb-11ea-90a7-ba9bfc053f16.png)
![image](https://user-images.githubusercontent.com/14900841/87755547-092dd880-c7bc-11ea-85f5-28797b834b4c.png)

Reply:
Can you turn off the linting on a per-line basis, instead of per-file?

Reply:
Tried this, and I still get the error. :(